corehq/apps/userreports/reports/filters/choice_providers.py

```diff
@@ -108,7 +108,7 @@ def _sql_column(self):
     def get_values_for_query(self, query_context):
         query = self._adapter.session_helper.Session.query(self._sql_column)
         if query_context.query:
-            query = query.filter(self._sql_column.contains(query_context.query))
+            query = query.filter(self._sql_column.ilike(u"%{}%".format(query_context.query)))
```

Comment:
(No blocker) Using `icontains` will give same result without having to use match strings.